src/theme/components/list.ts

```diff
@@ -10,7 +10,7 @@ const List = {
     padding: noStyling ? "0" : null,
     display: inline ? "flex" : null,
     li: {
-      marginRight: inline ? "m" : null,
+      marginRight: inline ? "xs" : null,
```

Comment:
When I noticed the other day, I did not realize this was a general style for an inline `List`.  That may have been why it was bumped up when you were initially working on the `List` inline styles.  If this is what it was originally, then we’ll leave it be for now and see if there is a use-case for increased spacing.  If that does come up, then we can write styles specific to `List` to control the spacing.